HeimdallrTests/Core/HeimdallrSpec.swift

```diff
@@ -544,6 +662,8 @@ class HeimdallrSpec: QuickSpec {
                         }, withStubResponse: { request in
                             return OHHTTPStubsResponse(data: NSData(contentsOfFile: self.bundle.pathForResource("request-valid", ofType: "json")!)!, statusCode: 200, headers: [ "Content-Type": "application/json" ])
                         })
+                        
+                        accessTokenParser.reset()
```

Comment:
@bckr: the test succeeds because the first request (that returns an expired access token) already triggered the flag. The reset is there to ensure that the parser is called a second time with the second request. If this is too fragile I can change the flag for a counter.